framework/decode/custom_vulkan_struct_decoders.cpp

```diff
@@ -563,5 +563,103 @@ size_t DecodeStruct(const uint8_t* buffer, size_t buffer_size, Decoded_VkImageTo
     return bytes_read;
 }
 
+size_t DecodeStruct(const uint8_t* buffer, size_t buffer_size, Decoded_VkDescriptorDataEXT* wrapper)
+{
+    GFXRECON_ASSERT((wrapper != nullptr) && (wrapper->decoded_value != nullptr));
+
+    size_t               bytes_read = 0;
+    VkDescriptorDataEXT* value      = wrapper->decoded_value;
+
+    bytes_read += ValueDecoder::DecodeUInt64Value(
+        (buffer + bytes_read), (buffer_size - bytes_read), &(value->accelerationStructure));
+
+    return bytes_read;
+}
```

Comment:
Why does this struct need a customized decode?  This seems like what would have been generated  otherwise.